dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/LambdaEventDAO.java

```diff
@@ -121,18 +177,20 @@ public List<LambdaEvent> findByOrganization(String organization, String offset,
      * @param repositories optional list of repositories
      * @return count of lambda events
      */
-    public long countByOrganization(String organization, Optional<List<String>> repositories) {
+    public long countByOrganization(String organization, Optional<List<String>> repositories, String filter) {
         CriteriaBuilder cb = currentSession().getCriteriaBuilder();
         CriteriaQuery<Long> query = cb.createQuery(Long.class);
         Root<LambdaEvent> event = query.from(LambdaEvent.class);
+
+        List<Predicate> initialPredicate = processQuery(filter, "", "", cb, query, event);
```

Comment:
Might be better to use a plural name (`initialPredicates`) since it's a `List` of `Predicates`.